packages/document-service-api/src/controllers/documents.js

```diff
@@ -1,21 +1,21 @@
 const {
-	initContainerClient,
 	downloadFile,
 	uploadFile,
 	deleteFile,
 	getMetadataForAllFiles,
 	getMetadataForSingleFile
 } = require('../lib/blobStorage');
 const deleteLocalFile = require('../lib/deleteLocalFile');
-const { documentTypes } = require('@pins/common');
+const { documentTypes, initContainerClient } = require('@pins/common');
 const { isFeatureActive } = require('../configuration/featureFlag');
 const logger = require('../lib/logger');
+const config = require('../configuration/config');
 
 const getDocumentsForApplication = async (req, res) => {
 	const { applicationId } = req.params;
 
 	try {
-		const containerClient = await initContainerClient();
+		const containerClient = await initContainerClient(config);
```

Comment:
`initContainerClient` from common now takes config as an arg so it's easier to use in different packages